modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -298,4 +372,31 @@ static JdbcDatabaseContainer getContainer(String jdbcUrl) {
             return jdbcUrlContainerCache.get(jdbcUrl);
         }
     }
+    
+    /**
+     * Register JDBC alias for custom JDBC-URL 
+     * @param alias		name of the alias e.g "my-own-mysql"
+     * @param type		driver type of the alias specified e.g. "mysql", "sqlserver", ...
+     * @param image		string that represents the docker image e.g. "mysql", "customregistry/image name"
+     * @param tag		string that represents the docker image tag e.g. "mytag", "4711", ..
+     */
+    public static void registerAlias(String alias, String type, String image, String tag) {		
+    	registerAlias(alias, type, image);    	
+    	ContainerDatabaseDriver.jdbcAliasMap.get(alias).put("tag", tag);			
+	}
+    
+    /**
+     * Register JDBC alias for custom JDBC-URL 
+     * @param alias		name of the alias e.g "my-own-mysql"
+     * @param type		driver type of the alias specified e.g. "mysql", "sqlserver", ...
+     * @param image		string that represents the docker image e.g. "mysql", "customregistry/image name"
+     */
+    public static void registerAlias(String alias, String type, String image) {		
```

Comment:
since we discourage the use of `:latest`, we should not provide any tag-less API

Reply:
@bsideup 
this will not fallback to latest but to the tag, that was specified by the module maintainer in the underlying specific container class.
The usecase here is when you host your own proxy registry, but want to use the tag, that was specified already.  Somewhat related to feture request in #2100 at least for the db containers